.gitignore

```diff
@@ -71,6 +71,12 @@ asset/dist
 
 # Build artifacts
 asset.zip
-build/
```

Comment:
We need to keep the build folder in the gitignore

Reply:
that was a duplicate, we still have build/ at line 64 of the file